src/services/pocket-relayer.ts

```diff
@@ -479,261 +494,269 @@ export class PocketRelayer {
     // Checks pass; create AAT
     const pocketAAT = new PocketAAT(...aatParams)
 
-    let node: Node
-
     // Pull the session so we can get a list of nodes and cherry pick which one to use
     const pocketSession = await this.pocket.sessionManager.getCurrentSession(
       pocketAAT,
       blockchainID,
-      this.pocketConfiguration
+      this.updateConfigurationTimeout(this.pocketConfiguration),
+      2
     )
 
-    if (pocketSession instanceof Session) {
-      // Start the relay timer
-      const relayStart = process.hrtime()
+    if (pocketSession instanceof Error) {
+      logger.log('error', 'ERROR obtaining a session: ' + pocketSession.message, {
+        relayType: 'APP',
+        typeID: application.id,
+        origin: this.origin,
+        blockchainID,
+      })
 
-      let nodes: Node[] = pocketSession.sessionNodes
+      return pocketSession
+    }
 
-      // sessionKey = "blockchain and a hash of the all the nodes in this session, sorted by public key"
-      const sessionKey = hashBlockchainNodes(blockchainID, nodes)
+    // Start the relay timer
+    const relayStart = process.hrtime()
 
-      this.pocketSession = pocketSession
-      const sessionCacheKey = `session-${sessionKey}`
+    let nodes: Node[] = pocketSession.sessionNodes
 
-      const nodesToRemove = await this.redis.smembers(sessionCacheKey)
+    // sessionKey = "blockchain and a hash of the all the nodes in this session, sorted by public key"
+    const sessionKey = hashBlockchainNodes(blockchainID, nodes)
 
-      if (nodesToRemove.length > 0) {
-        nodes = nodes.filter((n) => !nodesToRemove.includes(n.publicKey))
-      }
+    this.pocketSession = pocketSession
+    const sessionCacheKey = `session-${sessionKey}`
 
-      if (nodes.length === 0) {
-        logger.log('warn', `SESSION: blockchainHash: ${sessionKey} has exhausted all node relays`, {
-          requestID: requestID,
-          relayType: 'APP',
-          typeID: application.id,
-          serviceNode: '',
-        })
-        return new Error("session doesn't have any available nodes")
-      }
+    const nodesToRemove = await this.redis.smembers(sessionCacheKey)
+
+    if (nodesToRemove.length > 0) {
+      nodes = nodes.filter((n) => !nodesToRemove.includes(n.publicKey))
+    }
 
-      let syncCheckPromise: Promise<Node[]>
-      let syncCheckedNodes: Node[]
+    if (nodes.length === 0) {
+      logger.log('warn', `SESSION: blockchainHash: ${sessionKey} has exhausted all node relays`, {
```

Comment:
I'd prefer leaving it as before, blockchainHash in the string doesn't provide much value in that log.

`SESSION: ${sessionKey} blah blah`